developer-motivator/functions/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const admin = require('firebase-admin');
+const functions = require('firebase-functions');
+admin.initializeApp(functions.config().firebase);
+
+exports.appinstalled = functions.analytics.event('first_open').onLog(event => {
+    const payload = {
+      notification: {
+        title: 'new ' + event.data.user.deviceInfo.mobileModelName + ' from ' + event.data.user.geoInfo.city + ', ' + event.data.user.geoInfo.country,
+        body: 'new ' + event.data.user.deviceInfo.mobileModelName + ' from ' + event.data.user.geoInfo.city + ', ' + event.data.user.geoInfo.country
+      }
+    };
+
+     admin.messaging().sendToDevice("put_your_developer_device_token_here", payload).then(response => {
```

Comment:
Instead of `"put_your_developer_device_token_here"` could you use a config attribute for instance like `functions.config().dev_motivator.device_token`? And add a step in the readme to set it up before deploying please?